packages/@webex/plugin-cc/src/services/core/WebSocket/types.ts

```diff
@@ -1,5 +1,5 @@
 import {SubscribeRequest} from '../../../types';
-import {WebSocketManager} from './WebSocketManager';
+import {WebSocketManager} from './WebsocketManager';
```

Comment:
same comment here

Reply:
last set of my changes didn't get pushed, now you should see it